jsonnet/utils/set-terminationMessagePolicy.libsonnet

```diff
@@ -0,0 +1,42 @@
+{
+  setTerminationMessagePolicy(o): o {
+    local addTerminationMessagePolicy(o) = o {
+      [if std.setMember(o.kind, ['DaemonSet', 'Deployment', 'ReplicaSet']) then 'spec']+: {
```

Comment:
(nit) we shouldn't have any ReplicaSet object?

Reply:
yes I'm aware, though I figured we might as well throw it in their on the off change that we add one later.
The same is true for `initContainers`, we only have one in the node_exporter daemonset.

Reply:
ReplicaSet is a low-level object like Pod which I don't expect CMO to ever manage. But let's clean it up afterwards since it's also present in other jsonnet utilities.